src/select-menu/select-menu.scss

```diff
@@ -199,9 +201,8 @@
 // The icon shown on the left of a list item. Typically a check icon.
 
 .SelectMenu-icon {
-  position: absolute;
-  height: 1.5em; // Mimics line-height to vertically center the icon
-  margin-left: -20px;
+  width: 16px; // fixed width to make sure following content aligns
```

Comment:
My thinking was that the fixed width has less to do with "spacing" and is more related to the maximal width of Octicons, which are `16px`.

But yeah, I'll change it to `$spacer-3` to avoid ✨ magic numbers.